common/cards/hermits/monkeyfarm-rare.ts

```diff
@@ -0,0 +1,91 @@
+import {CardPosModel} from '../../models/card-pos-model'
+import {GameModel} from '../../models/game-model'
+import {discardCard} from '../../utils/movement'
+import HermitCard from '../base/hermit-card'
+import { getNonEmptyRows } from '../../utils/board'
+
+class MonkeyfarmRareHermitCard extends HermitCard {
+	constructor() {
+		super({
+			id: 'monkeyfarm_rare',
+			numericId: 157,
+			name: 'Monkeyfarm',
+			rarity: 'rare',
+			hermitType: 'farm',
+			health: 250,
+			primary: {
+				name: 'Skull',
+				cost: ['farm'],
+				damage: 50,
+				power: null,
+			},
+			secondary: {
+				name: 'Monkesystep',
+				cost: ['farm', 'farm'],
+				damage: 80,
+				power:
+					'Discard an item card from an AFK Hermit.',
```

Comment:
This needs' to say opponen't afk hermit. Again, needs balancing. Also gives me the idea of a hermit that could potentially discard one of our own afk hermits cards, but have a stronger attack as a result